website/docs/developer/deployment-guide/deploying-cumulus-dashboard-from-mac.md

```diff
@@ -0,0 +1,117 @@
+---
+id: deploying-from-mac
+title: Deploy Cumulus dashboard from Mac
+description: A concrete set of instructions on deploying Cumulus dashboard from Mac to create a test environment for ORCA. Also includes instructions for deploying the dashboard with launchpad authentication.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The goal of this page is to provide a set of additional instructions that will be useful to deploy the dashboard locally from Mac for ORCA project. It is assumed that Cumulus and ORCA are already deployed before following these instructions. Follow instructions on the following link on how to deploy Cumulus and ORCA:
+
+ - https://nasa.github.io/cumulus/docs/deployment/deployment-readme
+ - https://nasa.github.io/cumulus-orca/docs/developer/deployment-guide/deployment-with-cumulus
+
+## Setting up tunneling to locally run the dashboard
+
+There are several steps involved to set up tunnseling to use the dashboard locally.
+
+- Create a pac file named `aws-proxy.pac` and add the following function to that file and keep it under your `/.ssh` directory.
```

Comment:
Reorder to specify creation location when creating the file, and move "the following function" to the function.

Reply:
modified